packages/web/e2e/pages/wallet-page.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, Locator, Page } from "@playwright/test";
+
+export class WalletPage {
+  readonly page: Page;
+  readonly importWalletBtn: Locator;
+  readonly useRecoveryBtn: Locator;
+  readonly privateKeyBtn: Locator;
+  readonly privateKeyInput: Locator;
+  readonly importBtn: Locator;
+  readonly walletNameInput: Locator;
+  readonly walletPassInput: Locator;
+  readonly walletRePassInput: Locator;
+  readonly nextBtn: Locator;
+  readonly allCheckbox: Locator;
+  readonly saveBtn: Locator;
+  readonly finishBtn: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.importWalletBtn = page.getByText("Import an existing wallet");
+    this.useRecoveryBtn = page.getByText("Use recovery phrase or private key");
+    this.privateKeyBtn = page.getByRole("button", {
+      name: "Private key",
+      exact: true,
+    });
+    this.privateKeyInput = page.locator('input[type="password"]');
+    this.importBtn = page.getByRole("button", { name: "Import", exact: true });
+    this.walletNameInput = page.locator('input[name="name"]');
+    this.walletPassInput = page.locator('input[name="password"]');
+    this.walletRePassInput = page.locator("input[name='confirmPassword']");
+    this.nextBtn = page.getByRole("button", { name: "Next", exact: true });
+    this.allCheckbox = page.locator('input[type="checkbox"]').first();
+    this.saveBtn = page.getByRole("button", { name: "Save", exact: true });
+    this.finishBtn = page.getByRole("button", { name: "Finish", exact: true });
+  }
+
+  async goto(extensionId: string) {
+    await this.page.goto(`chrome-extension://${extensionId}/register.html#`);
```

Comment:
Does the playwright browser come preinstalled with Keplr?

Reply:
Unfortunately not, you need to build the extension and load it to the browser.

Reply:
added extension